setup.py

```diff
@@ -5,7 +5,8 @@
     description="Universal URL generator.",
     version='1.0.0',
     license='Apache',
+    author='Heureka.cz',
     author_email='vyvoj@heureka.cz',
-    url='git@github.com:heureka/py-url-generator.git',
+    url='https://github.com/heureka/py-url-generator',
```

Comment:
Odsouhlasilo se, ze ty nazvy budou bez prefixu `py-`. Nezmenime to rovnou? :)

Reply:
1) In English on Github please. :)
2) There are three different names for Python libraries:
a) Repo name – which is meaningful to have prefixed with `py-`, otherwise you couldn't upload `url-generator` for another language to the same Github organisation.
b) Library name – this is what we voted about – it is used for referencing the library using Python tools like `pip install`.
c) Package name – this is used when importing the library in code, so for example there must not be any dashes in the name (often replaced by underscores).